pyaqueduct/api.py

```diff
@@ -88,6 +88,17 @@ def get_experiment(self, alias: str) -> Experiment:
             created_at=experiment_data.created_at,
         )
 
+    @validate_call
+    def remove_experiment(self, alias: str) -> None:
```

Comment:
May I suggest to follow javascipt-ish way and call it `remove_experiment_by_alias()` or, looking in future, `remove_experiment_by_eid()`. Because  `remove_experiment(str)` to me looks dangerously ambiguous.

Reply:
Yeah make sense, I am going to update it.